packages/web/src/components/data-entry/FormTile.js

```diff
@@ -766,6 +792,10 @@ class FormTile extends Component {
           onSelectAllowAttribution={this.onSelectAllowAttribution}
           onSelectCommercialUse={this.onSelectCommercialUse}
           onSelectDerivativeWorks={this.onSelectDerivativeWorks}
+          remixSettingsModalVisible={remixSettingsModalVisible}
```

Comment:
I think only `setRemixSettingsModalVisible` is used in `AdvancedForm`, if my eyes are correct

Reply:
dang! eagle eyes